### PR TITLE
docs(sdk): Update render blocking status.

### PR DESCRIPTION
**Pull request by k-fish**
Adds mention of new data on resource spans, see https://github.com/getsentry/sentry-javascript/pull/7127

### CODE REVIEW

**Review by DominikB2014 — APPROVED** [review]

File: src/docs/sdk/performance/span-data-conventions.mdx [hunk 1]
```diff
@@ -19,11 +19,12 @@ Below describes the conventions for the Span interface for the `data` field on t
...
-| `Transfer Size`     | number | The transfer size of the request.          | `789`                 |
+| Attribute                | Type   | Description                                 | Examples              |
+| ------------------------ | ------ | ------------------------------------------- | --------------------- |
+| `url`                    | string | The URL of the resource that was fetched.   | `https://example.com` |
+| `method`                 | string | The HTTP method used.                       | `GET`                 |
+| `type`                   | string | The type of the resource that was fetched.  | `xhr`                 |
+| `Encoded Body Size`      | number | The encoded body size of the request.       | `123`                 |
+| `Decoded Body Size`      | number | The decoded body size of the request.       | `456`                 |
+| `Transfer Size`          | number | The transfer size of the request.           | `789`                 |
+| `Render Blocking Status` | number | The render blocking status of the resource. | `non-blocking`        |
```
Comment (DominikB2014) [inline src/docs/sdk/performance/span-data-conventions.mdx:30]: So the type of the `Render Block Status` is number, but the example is `non-blocking` (a string)

Comment (k-fish) [reply src/docs/sdk/performance/span-data-conventions.mdx:30]: Ah no it's supposed to be `string`, I thought I changed that. 👍

**Review by DominikB2014 — APPROVED** [review]

File: src/docs/sdk/performance/span-data-conventions.mdx (hunk 1 above)
Comment (DominikB2014) [reply src/docs/sdk/performance/span-data-conventions.mdx:30]: Awesome, all, lgtm once that's done!

Comment (AbhiPrasad) [inline src/docs/sdk/performance/span-data-conventions.mdx:30]:
```suggestion
| `resource.render_blocking_status` | string | The render blocking status of the resource. | `non-blocking`        |
```

**Review by AbhiPrasad — APPROVED** [review]

**Pull request merged**